common/core/gpudevice.cpp

```diff
@@ -201,6 +201,8 @@ void GpuDevice::DisplayManager::HotPlugEventHandler() {
           "UpdateDisplayState.");
       UpdateDisplayState();
     }
+    /* Adding sleep to avoid busy looping before a retry */
+    usleep(1000);
```

Comment:
hmm we shouldnt busy loop here, we should return when we have nothing to read. Is sleep really needed here ?